packages/data-model/migrations/0004_audit_submitted_data.sql

```diff
@@ -10,10 +10,11 @@ CREATE TABLE IF NOT EXISTS "audit_submitted_data" (
 	"comment" varchar,
 	"dictionary_category_id" integer NOT NULL,
 	"entity_name" varchar NOT NULL,
-	"is_valid" boolean NOT NULL,
 	"last_valid_schema_id" integer NOT NULL,
 	"new_data" jsonb,
+	"new_data_is_valid" boolean NOT NULL,
```

Comment:
Since this is `NOT NULL`, What is the value here when `new_data` is null?

Reply:
in the case where `new_data` is null `new_data_is_valid` will be null. This is the case of Delete submitted data